diff --git a/components/content_settings/core/common/cookie_settings_base.h b/components/content_settings/core/common/cookie_settings_base.h
index 509fbf0fa3adb1a249c5bf56d1657a52571eebad..c416509a59128cf309a0374661e91b7dc0481ad1 100644
--- a/components/content_settings/core/common/cookie_settings_base.h
+++ b/components/content_settings/core/common/cookie_settings_base.h
@@ -170,6 +170,7 @@ class CookieSettingsBase {
   // access.
   static bool IsValidSettingForLegacyAccess(ContentSetting setting);
 
+  BRAVE_COOKIE_SETTINGS_BASE_H
  private:
   virtual ContentSetting GetCookieSettingInternal(
       const GURL& url,
